File: 1470-shuffle-the-array/1470-shuffle-the-array.cpp
class Solution {
public:
    vector<int> shuffle(vector<int>& nums, int n) {
        vector<int> res;
        int left = 0;
        int right = n;
        for(int i = 0 ; i < n; i++){
         res.push_back(nums[i]);
         res.push_back(nums[i + n]);
        }
        return res;
    }
};

File: 1470-shuffle-the-array/1470-shuffle-the-array.py
from typing import List

class Solution:
    def shuffle(self, nums: List[int], n: int) -> List[int]:
        answer = [0 for _ in range(len(nums))]

        for i in range(n):
            answer[2*i] = nums[i]
            answer[2*i+1] = nums[n+i]

        return answer